client/src/components/calendar/MyCalendar.jsx

```diff
@@ -22,6 +22,29 @@ const MyCalendar = (props) => {
       return []
     }
   }
+
+  if (props.viewingShared) {
```

Comment:
I see that there's an if statement to check if the user is viewing a shared calendar but the return itself doesn't seem any different than what is being returned by default. Could you explain this?